cli/internal/semver/semver.go

```diff
@@ -0,0 +1,49 @@
+package semver
```

Comment:
We use https://github.com/Masterminds/semver in our [backend repo FYI](https://github.com/cloudquery/cloud/blob/08e805674577b347a07b91bc155802c08c1f5bf9/go.mod#L17)